elements/power.lua

```diff
@@ -246,8 +261,13 @@ local Enable = function(self, unit)
 		-- For tapping.
 		self:RegisterEvent('UNIT_FACTION', Path)
 
-		if(power:IsObjectType'StatusBar' and not power:GetStatusBarTexture()) then
-			power:SetStatusBarTexture[[Interface\TargetingFrame\UI-StatusBar]]
+		if(power:IsObjectType'StatusBar') then
+			power.texture = power:GetStatusBarTexture() and power:GetStatusBarTexture():GetTexture() or [[Interface\TargetingFrame\UI-StatusBar]]
+			power:SetStatusBarTexture(power.texture)
+		end
+		
+		if(power.bg) then
+			power.bg.texture = power.bg:GetTexture()
```

Comment:
`:SetColorTexture(1, 1, 1)` would make `:GetTexture()` return "Color-ffffffff"
`:SetColorTexture(1, 0, 0)` -> "Color-ff00ff"
`:SetColorTexture(0.5, 0, 0)` -> "Color-8000ff"
`SetColorTexture(0.6, 0, 0)` -> "Color-9900ff"
`SetColorTexture(0.01, 0, 0)` -> "Color-300ff"

It will be challenging to extract the color from that